bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZA2JToZA2L2J_EWK_MLL50_Mjj120_5f_LO/ZA2JToZA2L2J_EWK_MLL50_Mjj120_5f_LO_run_card.dat

```diff
@@ -0,0 +1,270 @@
+  91.188    =  dsqrt_q2fact1     ! fixed fact scale for pdf1
+  91.188    =  dsqrt_q2fact2     ! fixed fact scale for pdf2
+  1.0    =  scalefact         ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+  0    =  ickkw             ! 0 no matching, 1 MLM, 2 CKKW matching
+  1    =  highestmult       ! for ickkw=2, highest mult group
+  1    =  ktscheme          ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+  1.0    =  alpsfact          ! scale factor for QCD emission vx
+  False    =  chcluster         ! cluster only according to channel diag
+  True    =  pdfwgt            ! for ickkw=1, perform pdf reweighting
+  5    =  asrwgtflavor      ! highest quark flavor for a_s reweight
+  True    =  clusinfo          ! include clustering tag in output
+  3.0    =  lhe_version        ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+  False    =  auto_ptj_mjj   ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0    =  bwcutoff       ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*************************************************************
+  True    =  cut_decays     ! Cut decay products 
+#*************************************************************
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+  0    =  nhel           ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  20.0    =  ptj        ! minimum pt for the jets 
+  20.0    =  ptb        ! minimum pt for the b 
```

Comment:
Let's not give additional cuts to the phase space since you already have hard requirement from mjj. I don't think cross sections will be diverging